reference/feature-toggles.md

```diff
@@ -19,7 +19,8 @@ with a `boolean` value.
 
 * **`enableHackathonCasesPage`**
 
-  The Valtimo product development team participated in a Hackathon and developed a special cases page for municipalities.
+  The Valtimo product development team participated in a Hackathon and developed a special cases page for
```

Comment:
I know it was already there, but it doesn't seem relevant that we participated in a Hackaton.

Reply:
It is an existing feature flag. But I guess we could remove it from the FE code as well